client/apollo/react/src/Accordion/AccordionCommon.tsx

```diff
@@ -47,12 +48,13 @@ export const AccordionCommon = ({
   AccordionCoreComponent,
   TagComponent,
   IconComponent,
+  isPlain,
   ...accordionCoreProps
 }: AccordionCommonProps) => (
   <AccordionCoreComponent
     className={getClassName({
       baseClassName: "af-apollo-accordion",
-      modifiers: [variant],
+      modifiers: [variant, isPlain ? "plain" : undefined],
```

Comment:
C'est pas plus comme ca ?

```suggestion
      modifiers: [variant, isPlain && "plain"],
```

Reply:
done